src/kernel/src/fs/file.rs

```diff
@@ -226,6 +226,9 @@ pub trait FileBackend: Debug + Send + Sync + 'static {
         Err(Box::new(DefaultError::IoctlNotSupported))
     }
 
+    #[allow(unused_variables)]
+    fn poll(self: &Arc<Self>, file: &VFile, events: PollEvents, td: &VThread) -> PollEvents;
+
```

Comment:
Can we put the name on FreeBSD in its documentation so it is easier to follow the code?

Reply:
What do you mean? Like 'fo_poll'?

Reply:
Something like:

```rust
/// An implementation of `fo_poll`.
```